docs/getting_started/examples.rst

```diff
@@ -0,0 +1,67 @@
+
+Examples and Resources 
+========================
+
+The earlier sections of have covered the basics of using Modin to speed up your pandas workflow. Here you can find additional resource to learn about Modin. To learn more about advanced usage for Modin, please refer to :doc:`this section </advanced_usage/index>`.
```

Comment:
Seems to be some words missing here: "The earlier sections of [...] have covered". Also grammar nit in the second sentence, I believe "resource" should be made plural.

Reply:
Nice catch! Fixed now!